docs/root/start/sandboxes/jaeger_native_tracing.rst

```diff
@@ -0,0 +1,92 @@
+.. _install_sandboxes_jaeger_native_tracing:
+
+Jaeger Tracing
```

Comment:
Can you look at the rendered docs? There are a few oddities:
1) We show "Jaeger tracing" twice. Can you change the titles to make it more clear what they are about?
2) Some of the links are not properly linkified (I think missing '_' at the end of external links). This I think is a problem in both the old Jaeger stuff and the new text.

Reply:
Should be fixed now. Thanks.